src/app/auth/interceptors/jwt.interceptor.ts

```diff
@@ -0,0 +1,32 @@
+import { HttpInterceptor } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { AuthService } from '../services/auth.service';
+
+import { HttpRequest, HttpHandler, HttpEvent } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { SessionStorageService } from '../services/session-storage.service';
+
+@Injectable()
+export class JwtInterceptor implements HttpInterceptor {
+  constructor(
+    private authService: AuthService,
+    private sessionStorageService: SessionStorageService
+  ) {}
+
+  intercept(
+    request: HttpRequest<any>,
+    next: HttpHandler
+  ): Observable<HttpEvent<any>> {
+    const isAuth = this.authService.isAuthorized;
+    const token = this.sessionStorageService.getToken();
+    if (isAuth && token) {
+      request = request.clone({
+        setHeaders: {
+          Authorization: token,
```

Comment:
Does it already include the following shape "Bearer ${token}"?
I believe the token is Bearer since it's JWT